src/design-system/file-upload/file-upload.css.ts

```diff
@@ -48,6 +48,7 @@ export const checkIconBox = style([
     h: '$24',
     fontSize: '$25',
     color: '$data_green',
+    lineHeight: undefined,
```

Comment:
I haven't see this yet in ui toolkit. Can you describe what it does?

Reply:
This is an attempt to align the ticker icon centrally with the text. Need to double check on storybook.

Reply:
but `undefined` resets, removes `lineHeight` prop?

Reply:
It worked on the original repo, it seems that it needs an explicit 0 here.